docs/examples.md

```diff
@@ -157,6 +157,7 @@ check_task:
   check_script: gradle check
   cleanup_before_cache_script:
     - rm -rf ~/.gradle/caches/$GRADLE_VERSION/
+    - rm -rf ~/.gradle/caches/transforms-1 ~/.gradle/caches/journal-1
```

Comment:
Here is a suggestion to have them separately:

```suggestion
    - rm -rf ~/.gradle/caches/transforms-1
    - rm -rf ~/.gradle/caches/journal-1
```

Reply:
Done!